specification/multicast-notification-service.yaml

```
@@ -286,35 +207,88 @@ paths:
-              
+                  type: "object"
+                  properties: 
+                    versionId: 
+                      type: "string"
+                      description: "The versionId/ETAG identifies the version of the resource."
+                    fullUrl: 
+                      type: "string"
+                      description: "The URL where the subscriber can retrieve the source of truth record for the patient or the full event payload."
+                    provenance: 
+                      type: "object"
+                      description: "The provenance describes which actor was responsible for the creation of the event."
+                      properties:
+                        name: 
+                          type: "string"
+                          description: "The GP Practice name"
+                        identifier: 
+                          type: "object"
+                          required:
+                          - "system"
+                          - "value"
+                          description: "Must follow this identifier system: https://fhir.nhs.uk/Id/nhsSpineASID and provide the ASID value and name of the system."
+                          properties : 
+                              system: 
+                                type: "string"
+                              value: 
+                                type: "string"
+            examples:
+              withoutData:
+                value:
+                  id: "236a1d4a-5d69-4fa9-9c7f-e72bf505aa5b"
+                  type: "pds-change-of-gp-1"
+                  subject:
+                    nhsNumber: "9912003888"
+                    familyName: "DAWKINS"
+                    dob: "2017-10-02"
+                  source:
+                    name: "NHS Digital"
+                    identifier:
+                      system: "https://fhir.nhs.uk/Id/nhsSpineASID"
+                      value: "477121000324"
+                  time: "2022-04-05T17:31:00.000Z"
+
+              withData:
+                value:
+                  id: "236a1d4a-5d69-4fa9-9c7f-e72bf505aa5b"
+                  type: "pds-change-of-gp-1"
+                  subject:
+                    nhsNumber: "9912003888"
+                    familyName: "DAWKINS"
+                    dob: "2017-10-02"
+                  source:
+                    name: "NHS Digital"
+                    identifier:
+                      system: "https://fhir.nhs.uk/Id/nhsSpineASID"
+                      value: "477121000324"
+                  time: "2022-04-05T17:31:00.000Z"
+                  data:
+                    versionId: ' W/"2" '
+                    fullUrl: "https://test-api.nhs.uk/personal-demographics/FHIR/R4/Patient/5323035280"
```

```suggestion
                    fullUrl: "https://int.api.service.nhs.uk/personal-demographics/FHIR/R4/Patient/9912003888"
```

My bad - I got the initial url wrong. It should be int.api... Also need to ensure NHS Number after the / matches the patient NHS Number in question.